app/build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.0.0"
+
+    defaultConfig {
+        applicationId "org.musicbrainz.mobile"
+        minSdkVersion 7
+        targetSdkVersion 17
+        versionCode 18
+        versionName "1.3.5"
+    }
+
+    sourceSets {
+        main {
+            java.srcDirs = ['src']
+            assets.srcDirs = ['assets']
+            res.srcDirs = ['res']
+            manifest.srcFile 'AndroidManifest.xml'
+        }
+        androidTest.setRoot('../instrumentation')
+        androidTest {
+            java.srcDirs = ['../instrumentation/src']
+            manifest.srcFile '..instrumentation/AndroidManifest.xml'
+        }
+
+    }
+
+    android {
+        lintOptions {
+            abortOnError false
+        }
+    }
+}
+
+dependencies {
+    compile project(':api')
+    compile rootProject.files('deployer/PayPal_MPL-1.5.6.46.jar')
```

Comment:
This can move to the app project. Just create a new dir called `local` or similar.